meson.build

```diff
@@ -314,7 +314,7 @@ if has_semihost
   # If we're using semihosting, we assume that there's a
   # custom linker script that includes picolibc.specs
 
-  test_link_name = 'test-' + host_machine.cpu() + '.ld'
+  test_link_name = 'test-' + host_machine.cpu_family() + '.ld'
```

Comment:
This should probably be using the host_cpu_family computed above instead of just calling cpu_family here.

Reply:
Done